tests/drivers/regulator/pca9420/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 
 cmake_minimum_required(VERSION 3.20.0)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project(tests_drivers_regulator_pmic)
+project(tests_drivers_regulator_pca9420)
```

Comment:
Why move this test to be specific to the PCA9420? Is there something here that wouldn't work for another I2C compatible PMIC like the nPM6001?

Reply:
I'm planning to improve tests as well, but right now, it is too specific IMO.